app/league/slack_messenger.py

```diff
@@ -28,6 +26,9 @@ def init_app(self, app):
 
     def update_configuration(self, config):
         """Update Slack Messenger configuration."""
+        assert 'enabled' in config
```

Comment:
Well, here this'll raise a runtime exception, which may be valuable. However, it's not clear to me what the purpose is. To disable config updates if the messenger is disabled?